scriptworker/cot/verify.py

```diff
@@ -686,8 +686,9 @@ def _sort_dependencies_by_name_then_task_id(dependencies):
 
     if path not in link.cot['artifacts']:
         raise CoTError("path {} not in {} {} chain of trust artifacts!".format(path, link.name, link.task_id))
+    # Don't log the url, which may be signed/private
+    log.info("Downloading Chain of Trust artifact:\n{} {}".format(task_id, path))
```

Comment:
Curiosity: Out of curiosity, why don't we out `loggable_url` in this context?

Reply:
Sure. I think it's because I made this change before I wrote `get_loggable_url` :)